app/models/enterprise.rb

```diff
@@ -205,6 +207,13 @@ class Enterprise < ApplicationRecord
     ", one, one, others)
   }
 
+  def empty_field(attributes)
```

Comment:
The name of this method is very general but applies only to business addresses. Maybe `business_address_empty?` could be better?